docs/asciidoc/coreFeatures.adoc

```diff
@@ -457,7 +457,7 @@ Flux.just(10)
 ----
 ====
 
-You also have the option of applying a `Predicate` on the exception to decided
+You also have the option of applying a `Predicate` on the exception to decide
 whether or not to recover, as the folloiwng example shows:
```

Comment:
Aw man, I didn't notice a typo on the next line! `folloiwng`  :(

Reply:
Could fix two typos in one sentence at once! Well, maybe next time...

Reply:
ah snap XD
looks like a failed search-and-replace... maybe there are other occurrences in more files (including non-asciidoc ones)

Reply:
ah sorry, misinterpreted the comment... distinct typo, got it